content/academy/apify_platform/actors.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Actors
+description: What is an actor? How do we create them? Learn the basics of what actors are, how they work, and try out an actor yourself right on the Apify platform!
+menuWeight: 5.3
+paths:
+- apify-platform/actors
+---
+
+# [](#actors) Actors
+
+After you've followed the **Getting started** and **Apify CLI** lessons, and familiarized yourself with the Apify SDK in [**Web scraping for beginners**]({{@link web_scraping_for_beginners/crawling/pro_scraping.md}}), you're almost ready to start creating actors! But before we get into that, let's discuss what an actor is, and a bit about how they work.
```

Comment:
I think the CLI is not placed well in the course. You first need to know what an actor is before using the CLI.

Also 

> and familiarized yourself with the Apify SDK in [**Web scraping for beginners**]({{@link web_scraping_for_beginners/crawling/pro_scraping.md}})

This should not be needed at all. We discussed this yesterday.

Reply:
Swapped the CLI and Actor lessons for now, but there will be more in between them